test/screenshot/config.js

```diff
@@ -324,3 +328,216 @@ for (let color of getButtonConfig('paypal', 'colors')) {
         }
     });
 }
+
+
+/** INSTALLMENT BUTTON CONFIG **/
```

Comment:
For screenshots, added the config for installment btn separately. My early plan was to hijack the buttonConfig object and add the locale prop to button. However, our new friend flow doe not like adding properties to `sealed objects`. SO, I have duplicated the tests. COuld have done a bunch of ifs and lese in the existing one but thought of keep it separate and clean.
Creating unsealed objects would have been too much explicit declarations which we dont want.